src/tiny-bang-parser/tiny_bang_generated_lexer.mll

```diff
@@ -6,7 +6,7 @@ let digit = ['0'-'9']
 let alpha = ['a'-'z'] | ['A'-'Z']
 let whitespace = [' ' '\t' '\n']
 let comment = '#' [^'\n']* '\n'
-let integer = '-'? ['0'-'9']+
+let integer =  '-'? ('0' | ['1'-'9'] (digit | '_')*)
```

Comment:
Since you're so careful about whitespace, I might just remind you that there is a extra space before the `'-'?`. It doesn't even help the line align with any other line.